### ISSUE TITLE
Multi day events disappear

### ISSUE THREAD

**Issue by atheck** [aldabil21/react-scheduler#389]
I have multiple events and one multi day event. This multi day event renders at first, but after some milliseconds it disappears again.

![Calendar](https://github.com/user-attachments/assets/75ddc89d-f57c-4806-ab88-a9b169d11abf)

This only happens when I set the `weekStartOn` property to a non-default value, e.g. 1. With the default value the event stays visible.

This does not work:

```tsx
<Scheduler
  view={"week"}
  selectedDate={new Date(selectedDateIso)}
  week={{
    weekStartOn: 1,
    weekDays: [0, 1, 2, 3, 4, 5],
    startHour: 0,
    endHour: 24,
    step: 60,
  }}
  events={scheduleEvents}
```

Works:

```tsx
<Scheduler
  view={"week"}
  selectedDate={new Date(selectedDateIso)}
  week={{
    weekStartOn: 6,
    weekDays: [0, 1, 2, 3, 4, 5],
    startHour: 0,
    endHour: 24,
    step: 60,
  }}
  events={scheduleEvents}
```

**Comment by aldabil21** [comment]
I'm not quite sure that I've replicated that exact case 100%, but it might be solved in #393 

If not, please provide me with the minimum sample of your events array (start/end dates only)

The PR merge will auto-close this issue, feel free to reopen if it's still there

**Comment by atheck** [comment]
Thank you very much. I'll give it a try.
